aspnetcore/blazor/fundamentals/additional-scenarios.md

```diff
@@ -112,20 +112,62 @@ Blazor Server apps are set up by default to prerender the UI on the server befor
 
 Rendering server components from a static HTML page isn't supported.
 
-## Configure the SignalR client for Blazor Server apps
+## Manual start of the Blazor app
 
 *This section applies to Blazor Server.*
 
-Configure the SignalR client used by Blazor Server apps in the `Pages/_Host.cshtml` file. Place a script that calls `Blazor.start` after the `_framework/blazor.server.js` script and inside the `</body>` tag.
+Configure the manual start of a Blazor Server app's [SignalR circuit](xref:blazor/hosting-models#circuits) in the `Pages/_Host.cshtml` file:
 
-### Logging
+* Add an `autostart="false"` attribute to the `<script>` tag for the `blazor.server.js` script.
+* Place a script that calls `Blazor.start` after the `blazor.server.js` script's tag and inside the closing `</body>` tag.
 
-To configure SignalR client logging:
+When `autostart` is disabled, any aspect of the app that doesn't depend on the circuit works normally. For example, client-side routing is operational. However, any aspect that depends on the circuit isn't operational until `Blazor.start` is called. For example, component methods fail to execute while the circuit is disconnected.
```

Comment:
I like the idea of using an example here but I think it doesn't convey the severity of not having an initialized Blazor circuit enough. Maybe replace the example with a warning that application behavior will be unpredictable if the circuit is not trusted.